internal/commands/config/list_keys_internal.go

```diff
@@ -0,0 +1,106 @@
+package config_internal
+
+import (
+	"strings"
+
+	"github.com/pingidentity/pingcli/internal/configuration"
+	"github.com/pingidentity/pingcli/internal/configuration/options"
+	"github.com/pingidentity/pingcli/internal/logger"
+	"github.com/pingidentity/pingcli/internal/output"
+	"github.com/pingidentity/pingcli/internal/profiles"
+	"gopkg.in/yaml.v3"
+)
+
+func returnKeysYamlString() string {
+	inputString := returnKeysString("", " ")
+	// Split the input by spaces
+	parts := strings.Fields(inputString)
+
+	// Create a nested map based on the period (.) separator
+	result := make(map[string]interface{})
+
+	// Flag to indicate if activeProfile is processed
+	activeProfileFound := false
+
+	// Iterate through each part
+	for _, part := range parts {
+		keys := strings.Split(part, ".")
+
+		// Only treat the first occurrence of activeProfile as the root key
+		if keys[0] == "activeProfile" && !activeProfileFound {
```

Comment:
I think we need to remove this as a key output entirely.

the pingcli commands `list-profiles` and `set-active-profile` should handle activeProfile information and interaction.